### ISSUE TITLE
Fonts too big in 4K screen

### ISSUE THREAD

**Issue by Ricarp** [evanshultz/neater-bookmarks#90]
![screenshot 3](https://user-images.githubusercontent.com/33192036/32141020-f1620564-bc42-11e7-933f-23b7c438bb8d.jpg)

**Comment by alon-to** [comment]
Yeah, looks like this extension remains dead. Has promise and had a big user base but oh well. I've switched to [Popup My Bookmark](https://chrome.google.com/webstore/detail/popup-my-bookmarks/mppflflkbbafeopeoeigkbbdjdbeifni?hl=en) -- similar plug-in and works better in 4k.
